std/complex.d

```diff
@@ -292,14 +292,14 @@ struct Complex(T)  if (isFloatingPoint!T)
             // r = lhs
             // theta = 0
             ab = lhs ^^ this.re;
-            ar = log(lhs) * this.im;
+            ar = std.math.log(lhs) * this.im;
         }
         else
         {
             // r = -lhs
             // theta = PI
-            ab = (-lhs) ^^ this.re * exp(-PI * this.im);
-            ar = PI * this.re + log(-lhs) * this.im;
+            ab = (-lhs) ^^ this.re * std.math.exp(-std.math.PI * this.im);
+            ar = std.math.PI * this.re + std.math.log(-lhs) * this.im;
```

Comment:
Hmm,  you're right, `std.complex` quite possibly may implement its own version of `log`, `exp`, etc.. But wouldn't they be adequately disambiguated by the argument type? Or would that still be a conflict under current rules?